source/expressions.tex

```diff
@@ -2242,22 +2242,26 @@
 is said to be \defn{explicitly captured}.
 
 \pnum
-If an \grammarterm{identifier} in a \grammarterm{simple-capture} appears
```

Comment:
@jensmaurer: The paper says,

> The changes to this paragraph have not been explicitely discussed
by EWG, and may be better treated as a core issue or an NB comment.

Is CWG aware of this and has the straw poll motion been taken with this information well-known?

Reply:
CWG has seen the wording as we're seeing it now, so was aware of this.

(CWG chair) This feels like an obvious consistency fix to me that doesn't need EWG input.

Reply:
That's great, thanks!